Breeze.BreezeServer/Services/TumblerService.cs

```diff
@@ -27,11 +27,17 @@ public void StartTumbler(BreezeConfiguration breezeConfig, bool getConfigOnly, s
         {
             var argsTemp = new List<string>();
             argsTemp.Add("-debug");
-			
-			if (breezeConfig.TumblerNetwork == Network.TestNet)
-				argsTemp.Add("-testnet");
-			else if (breezeConfig.TumblerNetwork == Network.RegTest)
-			    argsTemp.Add("-regtest");
+
+            if (breezeConfig.TumblerNetwork == Network.TestNet)
+                argsTemp.Add("-testnet");
+            else if (breezeConfig.TumblerNetwork == Network.RegTest)
+                argsTemp.Add("-regtest");
+            else if (breezeConfig.TumblerNetwork == Network.StratisMain)
+                argsTemp.Add("-stratis");
+            else if (breezeConfig.TumblerNetwork == Network.StratisTest)
+                argsTemp.Add("-stratistest");
+            else if (breezeConfig.TumblerNetwork == Network.StratisRegTest)
+                argsTemp.Add("-stratisregtest");

```

Comment:
Feels like we need some sort of extension method for a Network enum so that we can put that logic into one place and replace all this code with just `argsTemp.Add(breezeConfig.TumblerNetwork.ToSuffix())` or something along those lines. Same for bitcoin network string above. Something like `bitcoinNetwork = TumblerNetwork.ToNamedString()` would be cleaner